File: src/main/java/fi/finlit/edith/ui/pages/Logout.java
/*
 * Copyright (c) 2009 Mysema Ltd.
 * All rights reserved.
 *
 */
package fi.finlit.edith.ui.pages;

import org.apache.tapestry5.ioc.annotations.Inject;

import fi.finlit.edith.ui.services.AuthService;

public class Logout extends Base {

    @Inject
    private AuthService authService;

    public void onActivate(){
        authService.logout();
    }

}

File: src/main/scala/fi/finlit/edith/ui/pages/Logout.scala
package fi.finlit.edith.ui.pages

import org.apache.tapestry5.ioc.annotations.Inject
import fi.finlit.edith.ui.services.AuthService
//remove if not needed
import scala.collection.JavaConversions._

class Logout extends Base {

  @Inject
  private var authService: AuthService = _

  def onActivate() {
    authService.logout()
  }
}
